modules/crud.js

```diff
@@ -0,0 +1,77 @@
+import Todo from './todo.js';
+
+const mainList = document.querySelector('.tasks');
+
+class TodoList {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('list')) || [];
+    this.displayList();
+    this.enterEvent();
+  }
+
+  add(text) {
+    const newTodo = new Todo(text);
+    this.list.push(newTodo);
+    localStorage.setItem('list', JSON.stringify(this.list));
+    this.displayList();
+  }
```

Comment:
- [ ] Great job working on all the functionality but you still have one thing to change when i see the tasks on localstorage I can't see that it has indexes so whenever the user remove the task the indexes should be updated as required ✅⬇🔥🤩 so if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1) so kindly implement the indexes here to each task 🔥💯


![image](https://user-images.githubusercontent.com/58553711/236086920-bb42c4c4-3224-4884-8b76-ed0f6631a5be.png)


- I can see just completed and the name of the task
![image](https://user-images.githubusercontent.com/58553711/236087830-cab3b9ec-51b1-4896-a380-954826a24511.png)